index.html

```diff
@@ -255,7 +255,7 @@ <h3>Binaries</h3>
                 </div>
                 <div class="col-lg-3 col-md-6 text-center">
                     <div class="feature-box-primary">
-                        <a target="_blank" href="http://docs.vavr.io">
+                        <a target="_blank" href="//docs.vavr.io">
```

Comment:
 ⛔️ https *not* supported

Reply:
Thats too bad. I wonder if that could be configured to support the vanity url.
An alternative (though, less likely desired) is to point to the resolved URL in the interim.
This came up for me when I was looking through the docs and Google Chrome made point of telling me that the site was "not secure".

For now I have reverted it.